Esri/arcgis-maps-sdk-swift-samples
# [Fix] Remove the relevant APIs from the tags in the `Sample` struct

## Description

This PR implements a simple fix to remove the relevant APIs from `tags` property in the `Sample` struct, so that only the sample's tags are included in the property. It does this by updating the code generation in `GenerateSampleViewSourceCode` for the sample structs found in `SamplesApp+Samples`.

## Linked Issue(s)

- `swift/issues/4398`

## How To Test

- Clean the build folder and build the project.
- Verify the sample structs `tags` property in `SamplesApp+Samples` do not include the `relevant_apis` found in the corresponding metadata file.
